hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/main/java/org/apache/hadoop/yarn/client/api/impl/AHSv2ClientImpl.java

```diff
@@ -137,9 +137,7 @@ public ContainerReport getContainerReport(ContainerId containerId)
     ApplicationId appId = applicationAttemptId.getApplicationId();
     ApplicationReport appReport = getApplicationReport(appId);
     Map<String, String> filters = new HashMap<>();
-    filters.put("infofilters", "SYSTEM_INFO_PARENT_ENTITY eq {\"id\":\"" +
-        applicationAttemptId.toString() +
-        "\",\"type\":\"YARN_APPLICATION_ATTEMPT\"}");
+    filters.put("appattemptId", applicationAttemptId.toString());
```

Comment:
For me, this part seems suspicious. I'm not an expert in ATSv2 but I don't see the reason what does "SYSTEM_INFO_PARENT_ENTITY eq something" mean and why it would be easily replacable with "appattemptId".
If we figure this out from the code, I can proceed with the review.
Alternatively, we could try this out on a working cluster.
@brumi1024 What do you think?

Reply:
@szilard-nemeth built the solution proposal, tested it, and it indeed doesn't work, the related ATS endpoint doesn't have a queryparam called [appattemptId](https://github.com/apache/hadoop/blob/18f959d56a6dafd045a54a9d669bb969305947d3/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-timelineservice/src/main/java/org/apache/hadoop/yarn/server/timelineservice/reader/TimelineReaderWebServices.java#L606).